browser/ui/webui/brave_wallet/android/android_wallet_page_ui.cc

```diff
@@ -77,6 +77,13 @@ AndroidWalletPageUI::AndroidWalletPageUI(content::WebUI* web_ui,
       std::string("frame-src ") + kUntrustedTrezorURL + " " +
           kUntrustedLedgerURL + " " + kUntrustedNftURL + " " +
           kUntrustedLineChartURL + " " + kUntrustedMarketURL + ";");
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::ImgSrc,
+      base::JoinString(
+          {"img-src", "'self'", "chrome://resources",
+           "chrome://erc-token-images", "chrome://favicon", "chrome://image",
+           "https://assets.cgproxy.brave.com", base::StrCat({"data:", ";"})},
```

Comment:
just to make sure that @diracdeltas is ok with any csp changes

Reply:
this was copied from https://github.com/brave/brave-core/blob/4a7de952e5c0087aa6a3a9b702e794b770ebdbed/browser/ui/webui/brave_wallet/wallet_panel_ui.cc#L80-L86 ? why was this part of the chromium update?

Reply:
i think we should remove https://assets.cgproxy.brave.com from this unless it's confirmed to be necessary

Reply:
Yes, it was excessive, pushed the fix https://github.com/brave/brave-core/commit/72026c23aa87a718720eb039c746053545f4ee6b